From 402cad1b6f5605ed854eb8b7b7376cafce3fb007 Mon Sep 17 00:00:00 2001
From: Andi Kleen <ak@linux.intel.com>
Date: Fri, 29 Apr 2016 17:50:25 -0700
Subject: [PATCH 11/18] oprofile: Add model numbers for Kabylake CPUs

The PMU is using the same events as Skylake, so no other changes.

Signed-off-by: Andi Kleen <ak@linux.intel.com>
---
 libop/op_hw_specific.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libop/op_hw_specific.h b/libop/op_hw_specific.h
index f4db8f5..2061760 100644
--- a/libop/op_hw_specific.h
+++ b/libop/op_hw_specific.h
@@ -157,6 +157,8 @@ static inline op_cpu op_cpu_specific_type(op_cpu cpu_type)
 		case 0x4e:
 		case 0x5e:
 		case 0x55:
+		case 0x8e:
+		case 0x9e:
 			return CPU_SKYLAKE;
 		case 0x37:
 		case 0x4d:
-- 
2.7.4

